plugins/excel/core/src/main/java/org/pentaho/di/trans/steps/excelwriter/ExcelWriterStep.java

```diff
@@ -134,17 +138,17 @@ public boolean processRow( StepMetaInterface smi, StepDataInterface sdi ) throws
 
         // Let's remember where the fields are in the input row
         int outputFieldsCount = meta.getOutputFields().length;
-        data.commentauthorfieldnrs = new int[ outputFieldsCount ];
-        data.commentfieldnrs = new int[ outputFieldsCount ];
-        data.linkfieldnrs = new int[ outputFieldsCount ];
-        data.fieldnrs = new int[ outputFieldsCount ];
+        data.commentauthorfieldnrs = new int[outputFieldsCount];
```

Comment:
I think we've gone back and forth on this formatting before. I don't remember what the conclusion was. Causes a lot of noise in the PR if nothing else.

Reply:
i'll try to avoid file-wide reformats, but once it's in it's very hard to undo